webapp/src/app/course-welcome/course-welcome.component.ts

```
@@ -149,6 +506,47 @@
+      });
+    }
+                
+    this.courseDescription = this.selectedCourse.description;
+  }
+
+  customImageUpload(editor: Quill): void {
+    this.fileInput.nativeElement.click();
+  }
+  onFileSelected(event: Event): void {
+    const input = event.target as HTMLInputElement;
+    if (input.files && input.files[0]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)